Block/Checkout/Onepage/Success/PaynowAdditionalInformation.php

```diff
@@ -31,7 +31,7 @@ protected function _toHtml()
     {
         $paymentData = $this->_checkoutSession->getLastRealOrder()->getPayment()->getData();
         if (!isset($paymentData['additional_information']['payment_type']) || $paymentData['additional_information']['payment_type'] !== 'paynow') {
-            //return;
+            return;
```

Comment:
would it be better if we move all the following code into this conditional block (and check if the condition is true instead)
```php
$orderCurrency = $this->_checkoutSession->getLastRealOrder()->getOrderCurrency()->getCurrencyCode();

$this->addData([
    'paynow_qrcode' => $paymentData['additional_information']['qr_code_encoded'],
    'order_amount' => number_format($paymentData['amount_ordered'], 2) .' '.$orderCurrency
]);
```

So we can keep `return parent::_toHtml();` at the end for both payment_type= 'paynow' and else cases